Dockerfile

```diff
@@ -117,8 +117,15 @@ RUN --mount=type=cache,target=/var/cache/apt --mount=type=cache,target=/var/lib/
 
 COPY --from=build /usr/local/src/ort/scripts/*.sh /opt/ort/bin/
 
+# This can be set to a directory containing CRT-Files for custom certificates that ORT and all build tools should know about.
```

Comment:
I really meant `CRT-files` with lowercase "f".

The commit message still has "folder":

> Note that the folder has to be inside the Docker build context.